config/checkstyle-examples-suppressions.xml

```diff
@@ -49,4 +49,8 @@
   <suppress checks="FileLength"
     files="visibilitymodifier[\\/]Example\d+.java"/>
 
+  <!-- Examples for understanding fallthrough cases -->
+  <suppress checks="FallThrough"
+    files="fallthrough[\\/]Example\d+.java"/>
+
```

Comment:
Are you sure that we need this ?
Examples are all about to make code bad on purpose, we should not even validate them at all.
Try remove, if there is violation, please share it.

Reply:
ok. working on it

Reply:
This is done.